src/sentry/api/event_search.py

```diff
@@ -571,13 +571,14 @@ def create_from(cls, search_config: SearchConfig, **overrides):
 class SearchVisitor(NodeVisitor):
     unwrapped_exceptions = (InvalidSearchQuery,)
 
-    def __init__(self, config=None, params=None, builder=None):
+    def __init__(self, config=None, params=None, builder=None, get_field_type=None):
```

Comment:
I see there's also a `get_function_result_type` on `builder` that's used here. What's the plan for that? Add another optional argument?

Reply:
🤔 I think i regret the pattern of passing the entire builder class in, so yeah I think so. I'll probably remove the builder param later too